### DISCUSSION TITLE
osu lazer fps locked to 158 suddenly

### DISCUSSION THREAD

**Discussion by pmeiggs** [ppy/osu#23190] [Q&A]
idk what happened. I just played normally and then after I finished a map my fps are suddenly locked to 158. I've got a 165hz monitor and all idk what's happening.

**Comment by Rekunan** [comment]
What do you have ur frame limiter set to? Also any logs you can submit (assuming ur still experiencing the issue) (#15188)?

**Reply by pmeiggs** [reply]
I solved it a few days after, the problem was something to do with putting the game on windowed instead of fullscreen.
I apologise for a late response.

**Reply by Rekunan** [reply]
No worries.
Yep, that'd do it, glad to hear you solved it yourself.
